packages/yoga/src/Datepicker/web/Calendar.jsx

```diff
@@ -205,19 +205,26 @@ function Calendar({
     return new Date(Date.UTC(year, month, day)).getTime();
   };
 
+  const isSameDate = (date1, date2) =>
+    date1.getDate() === date2.getDate() &&
+    date1.getMonth() === date2.getMonth() &&
+    date1.getFullYear() === date2.getFullYear();
+
   const isEqual = day => {
+    const utcDate = new Date(Date.UTC(year, month, day));
+
     return (
-      (startDate && toUTCDate(startDate) === toUTCDateDay(day)) ||
-      (endDate && toUTCDate(endDate) === toUTCDateDay(day))
+      (startDate && isSameDate(startDate, utcDate)) ||
+      (endDate && isSameDate(endDate, utcDate))
     );
```

Comment:
Same as above.

Reply:
It is not so simple move this function because of the dependencies( year, month, startDate and endDate ). Do you think it is worth make a refactor?

Reply:
Ok :) you can leave it like this